cmd/run.go

```diff
@@ -8,21 +8,18 @@ import (
 	"os"
 
 	"github.com/spf13/cobra"
-	"github.com/vmware-tanzu/crash-diagnostics/exec"
-	"github.com/vmware-tanzu/crash-diagnostics/parser"
-	"github.com/vmware-tanzu/crash-diagnostics/script"
+	"github.com/vmware-tanzu/crash-diagnostics/starlark"
 )
 
 type runFlags struct {
-	file   string
-	output string
+	file string
 }
 
 // newRunCommand creates a command to run the Diaganostics script a file
 func newRunCommand() *cobra.Command {
 	flags := &runFlags{
-		file:   "Diagnostics.file",
-		output: "out.tar.gz",
+		file: "Diagnostics.file",
+		//output: "out.tar.gz",
```

Comment:
nit: will finalize it, once we make the decision

Reply:
We made the decision to remove the `--output` flag and instead make provisions to send an arbitrary key-value argument pair via the `--args` flag.